client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -142,6 +142,8 @@ class CalypsoifyIframe extends Component<
 		MediaStore.on( 'change', this.updateImageBlocks );
 		window.addEventListener( 'message', this.onMessage, false );
 
+		window.performance.mark( 'iframe_mounted' );
```

Comment:
This should be something like `window.performance?.mark()`, I'll change later

Reply:
From a functionality standpoint, it's worth noting that I'd expect the "iframe_mounted" event to happen when we actually render an `<iframe />` for the first time.  However, if you take a look at the code below, this will happen only when `shouldLoadIframe || isIframeLoaded` is `true` for the first time.